### ISSUE TITLE
Fix OpenGraph data on "facebook share"-button

### ISSUE THREAD

**Issue by melpomene** [youtify/youtify#40]
Currently it only display info about Youitfy and not the shared track

**Comment by pthulin** [comment]
Is there a way to redirect the requesting server to the YouTube video URL? This way we wouldn't have to do the request from our server.

**Comment by melpomene** [comment]
Yes, I was mostly thinking about the name of the artist and the song and not the embedded video though. But yeah, sure it is possible, just put the youtube streaming info (the same you find on the youtube page) in your OpenGraph data.
